geocity/fields.py

```diff
@@ -41,6 +41,15 @@ def media(self):
         )
 
 
+class GeometryWidgetAdvanced(geoforms.OpenLayersWidget):
+    template_name = "advancedgeometrywidget/geometrywidget.html"
+    map_srid = 2056
+
+    @property
+    def media(self):
+        return forms.Media()
```

Comment:
Ajouter le JS à ce niveau là, plutôt que dans le template ? (geocity/apps/submissions/templates/advancedgeometrywidget/geometrywidget.html)
```suggestion
        return forms.Media(
                js=(
                       "geocity-wc-map/geocity-wc-map.js",
                       "js/advancegeometrywidget/geometrywidget.js",
                ),
        )
```

Reply:
Good point but loading order seems to cause an issue